lib/mix/lib/mix/tasks/format.ex

```diff
@@ -226,7 +231,7 @@ defmodule Mix.Tasks.Format do
     {formatter_opts_and_subs, _sources} =
       eval_deps_and_subdirectories(dot_formatter, [], formatter_opts, [dot_formatter])
 
-    find_formatter_and_opts_for_file(file, formatter_opts_and_subs)
+    find_formatters_and_opts_for_file(file, formatter_opts_and_subs)
```

Comment:
Does the pluralization of these functions add clarity or detract?

Reply:
I{ think it detracts specially because now it still returns a single `formatter` function.